docs/components/expression-metadata.js

```diff
@@ -121,11 +121,38 @@ const types = {
         {
             type: 'boolean',
             parameters: [
-                'needle: (boolean, string or number)',
-                'haystack: (array or string)'
+                'InputType (boolean, string, or number)',
```

Comment:
Following up from https://github.com/mapbox/mapbox-gl-js/pull/9655#issuecomment-626022534, how about 
```suggestion
                'searchValue (boolean, string, or number)',
                'input (array or string)'
```

cc @ryanhamley